### DISCUSSION TITLE
What's possible by the morning of Summer 3

### DISCUSSION THREAD

**Discussion by nathan-alden-sr** [nathan-alden-sr/stardew-valley#41] [Show and tell]
I've played through Summer 2 several times now, getting better each time. In my full Year 1 playthrough, I had reached 552 Starfruit Seeds. In my current playthrough I reached 548. I wanted to post a picture of my Farm on the morning of Summer 3. The design closely matches the [Starfruit crop planner](https://github.com/nathan-alden-sr/stardew-valley/blob/master/min-max/1.5/readme.md#crop-planning) from the guide. Most of the seeds on the left are Mixed Seeds.

![MinMax_5-24-2021_48377772](https://user-images.githubusercontent.com/1189759/119354482-218b4100-bc72-11eb-86e1-000e3c9b8489.png)

My total earnings are 401,381g and I've acquired the following:
- Galaxy Sword
- Iridium Band
- Lucky Ring
- Rusty Key
- Seven Tappers
- Seven Lightning Rods
- Barn
- Coop (under construction)

As usual, I only have a few Coal left.

**Comment by waynn** [comment]
This is actually super helpful, it's quite good to have things to hit. Really impressive you already have the barn and ... Stable?

What's in the top left corner?

When did you hit SC by? Spring 15?

Other random question I was about to post: Why does the first batch of Starfruit matter so much? If you do 100, 400, 400 or do 300,300,300, isn't it equivalent (except for needing more sprinklers in the first case)?

**Reply by nathan-alden-sr** [reply]
I have a Barn, plus a Coop under construction. The top-left is a generic Tree farm with leftover Seeds. Regarding the Tree farm, it's actually really, really important to get as many Trees as possible as early as possible for two reasons:

- Raising Foraging skill
- Providing enough Wood to construct many Buildings

In this playthrough, I was able to earn the 42,500g necessary by Spring 15 to unlock Skull Cavern at day's end.

As I've said elsewhere, it's all about daily Luck. Getting rainy days on _specific_ days in Spring is extremely important as Catfish make up a huge portion of income before Skull Cavern really gets rolling. If it doesn't rain much, or the rain is on the wrong days, it can greatly hamper income potential.

Also, I have save files for every single day of my playthrough so far. I'm currently on Winter 24. I plan on uploading these to the repo soon, plus posting the videos of each day to YouTube. It'll just take time to edit everything.

**Reply by nathan-alden-sr** [reply]
The first batch doesn't matter so much as long as you don't remove the dead Crops. However, for the purposes of me testing the limits and recording videos, I kept going until I could get 548 Starfruit planted on Spring 2.

Finding enough Ore, Coal, and Quartz to craft more Quality Sprinklers than I did (76) is really challenging. What you see in the screenshot is approaching what is realistically possible without doing something like Luck manipulation (playing every day over again until the following day is "very happy") or getting insanely good fishing luck and fishing up stuff like Treasure Chests.

**Reply by waynn** [reply]
I actually ended up buying a fair amount of Coal in this run. I didn't go to SC until Spring 18, but managed to plant 553, but also got lucky with 2 iridium sprinklers and 4 quality sprinklers from SC. One day I got 217 iridium ore plus 2 prismatic shards. I think if I'd gotten to SC earlier could have hit 600 pretty easily, but subsequent batches might be harder.

**Comment by waynn** [comment]
One other question, why didn't you spend more on Starfruit seeds, and instead chose to build the Barn/Coop?

**Reply by nathan-alden-sr** [reply]
In my previous playthrough, I made the mistake of waiting too long for Animal Buildings. Specifically, my Pig did not mature and find a Truffle before Fall ended. This time around, I made sure to get an early start so I could complete the Animal Bundle as soon as possible. In addition to earlier Bundle completion, it also had the benefit of giving me a good supply of Cheese for Skull Cavern runs. In the previous playthrough I was always struggling for food.

**Reply by waynn** [reply]
Hitting mushrooms every day can mitigate the food problem slightly.
